content/updates/2019-09-06-depreecation-policy.md

```diff
@@ -0,0 +1,38 @@
+
+Firstly, this is the formal announcement of our deprecation policy, found [here](). Our deprecation policy is dual-focused decision; a business decision on how we want to be consistent, and human decision on how we want our customers to trust us. We recommend you take some time to read up on the specific of the policy, and how it can impact your business.
+
+## Consistency
+
+As a hosting provider, consistency is very important, because that provides predictability. Thus far, over the years, we've had to deprecate several aspects of the system as technologies grow, change, or our dependencies get deprecated. Looking back, in every single instance of needing to deprecate something, the cloud.gov team has had to sit down and have a meeting on how we want to handle a specific deprecation. We discuss things like the impact to ourselves, customers, and technologies.
+
+We came to realise our deprecation policy had been, _what do we do this time?_, instead of following a policy which had been set. This provided an inconsistent focus on how we wanted to present news, information, and execution of the given deprecation. The presented both problems for us and customers, as we weren't consistent with messaging or execution, and customers were often not supported as well as they could have been through consistent messaging and consistent support.
+
+As a business unit, we came together and decided that was enough, that it's time to be better. So we came up with a formal deprecation policy for our business unit that other business units and agency can count on for consistentcy. This means there will not be any more inconsistencies about messaging, timelines, or deprecation support. Regardless of what we deprecate, we will be consistent about it, and that is how we feel we can best suit other business units and agencies.
+
+We are committed to leveraging this deprecation policy as needed, and our goal as a business unit is to be as consistent as possible when executing it.
+
+## Predictability
+
+Being consistent as a hosting provider also gives us predictability, which is the second facet of our focus. A [commercial study](https://science.sciencemag.org/content/327/5968/1018) has shown that human behaviour can be highly predictable, and we as humans running a business unit have decided we can leverage our consistency as a business unit to provide a comforting predictability to the humans running workloads on cloud.gov.
+
+Through our consistency in the new deprecation policy, we are going to be providing human-friendly predictability. While our business unit and agency customers are best suited on relying on our consistency, our human users are best suited through our predictability. We are commited that by not straying from our deprecation policy, our human users can predict our responses, timelines, and execution of the deprecation policy with ease.
```

Comment:
Last sentence is a bit unwieldy and non-conversational in the first half - perhaps rework to something like:

> We are committed to standardizing our deprecation policy so our human users can predict ...